cartography/intel/aws/ssm.py

```diff
@@ -31,14 +31,14 @@ def get_running_instance_ids(neo4j_session: neo4j.Session, region: str, current_
 @timeit
 @aws_handle_regions
 def get_instance_information(
-    boto3_session: boto3.session.Session, region: str, running_instance_ids: List[str],
+    boto3_session: boto3.session.Session, region: str, instance_ids: List[str],
 ) -> List[Dict[str, Any]]:
     client = boto3_session.client('ssm', region_name=region)
     instance_information: List[Dict[str, Any]] = []
-    for i in range(0, len(running_instance_ids), 50):
-        running_instance_ids_chunk = running_instance_ids[i:i + 50]
+    for i in range(0, len(instance_ids), 50):
+        instance_ids_chunk = instance_ids[i:i + 50]
         instance_information_chunk = client.describe_instance_information(
-            Filters=[{"Key": "InstanceIds", "Values": running_instance_ids_chunk}],
+            Filters=[{"Key": "InstanceIds", "Values": instance_ids_chunk}],
             MaxResults=50,
```

Comment:
50 might be too low to to give out NextToken, but to be safe we should still paginate this.

Reply:
@ramonpetgrave64 this is a bit of a weird API. It takes a list of instance IDs, and gives back a list of results for the instances. It seems that the maximum number of instance IDs it'll take in is 50, and the maximum results it'll give out is 50, so though it's pageable, I think effectively, it'll never page. I guess it doesn't hurt to be paranoid about it, and I can add a paginator in.